codex/erasure/erasure.nim

```diff
@@ -659,6 +669,44 @@ proc decode*(self: Erasure, encoded: Manifest): Future[?!Manifest] {.async.} =
 
   return decoded.success
 
+proc repair*(self: Erasure, encoded: Manifest): Future[?!void] {.async.} =
+  ## Repair a protected manifest slot
```

Comment:
This comment suggests we're repairing 1 slot in the protected manifest. But I think the code reconstructs the entire dataset. Otherwise, I would have expected a slotIndex to be a required argument.